### DISCUSSION TITLE
Dependency dashboard per base branch

### DISCUSSION THREAD

**Discussion by roded** [renovatebot/renovate#31014] [Suggest an Idea]
### Tell us more.

Hi,
Looking over the documentation, I don't think this is supported currently.

We have renovate configured to scan multiple baseBranches.
It would be great if it would be possible to configure renovate to create a separate dependency dashboard issue per base branch.

Cheers.

**Comment by HonkingGoose** [comment]
### Many dashboards, or one with sections?

For your feature, do you want:

- _Multiple_ Dependency Dashboard issues: one dashboard for each item in the `baseBranches` array?
- _One_ Dependency Dashboard issue: dashboard has sections for each item in the `baseBranches` array?

### One dashboard for me

I prefer one dashboard, because:

- I always know where to find the dashboard.
- The dashboard shows all things on one page.
- I can see patterns like "Dependency A was updated on `main`,  but not on `other-branch`.
- You can only pin _three_ issues per repository on GitHub, so many dashboard issues means you need to search to find the correct one.

**Reply by roded** [reply]
Multiple dashboard issues for me.
We have a monorepo containing several different projects. 
We also have multiple older patch branches for the repository which continue to need renovate updates.
With multiple baseBranches, the amount of noise in a single Dependency Dashboard issue is hard to handle.
I would like to be able to continually add and remove baseBranches as versions come in and out of support and manage their dependencies separately.

**Comment by roded** [comment]
Actually, I think it makes sense in general to allow splitting the Dependency Dashboard into multiple issues depending on various factors, per base branch (as requested above) and also per packageRule so that we could managed a dashboard per project in a monorepo.
